File: app/public_html/script/functions/samples/14.c
loop(){
    if (!isTargetVisible())
        getLowHp();
    else{
        if (abs(getHead() - getTargetHead()) >= 90){
            stepLeft();
            turnToTarget();
        }
        else
            attackRanged(getTargetX(), getTargetY());
    }
}

File: app/public_html/script/functions/samples/14.py
def loop():
    if not isTargetVisible():
        getLowHp()
    else:
        if abs(getHead() - getTargetHead()) >= 90:
            stepLeft()
            turnToTarget()
         
        else:
            attackRanged(getTargetX(), getTargetY())
     
 